cluster-autoscaler/version/version.go

```diff
@@ -17,4 +17,4 @@ limitations under the License.
 package version
 
 // ClusterAutoscalerVersion contains version of CA.
-const ClusterAutoscalerVersion = "1.23.0-alpha.0"
```

Comment:
Actually this is a version of Cluster Autoscaler, not the kubernetes vendored in. CA matches the minor release number with k8s, but not patch release numbers and since we're not releasing CA we should not be bumping this.

Reply:
Done, rollbacked this file.